builtins/src/cost_modeling.rs

```diff
@@ -0,0 +1,300 @@
+//! Configurations for handling cost modeling of builtin programs.
+
+use {
+    crate::BUILTINS,
+    ahash::AHashMap,
+    lazy_static::lazy_static,
+    solana_feature_set::FeatureSet,
+    solana_pubkey::Pubkey,
+    solana_sdk_ids::{ed25519_program, secp256k1_program},
+};
+
+// Hehe, if you change me, I will break things...
+pub const NUM_COST_MODELED_BUILTINS_WITH_MIGRATIONS: usize = 3;
```

Comment:
Yeah I'm sure you tried the same steps as me 😂 - I couldn't get a solution where there wasn't some compiler gotcha, but at least here it will throw if you don't do it right.